conf.yaml

```diff
@@ -420,8 +420,8 @@ contents:
             prefix:     en/cloud
             tags:       Cloud/Reference
             subject:    Elastic Cloud
-            current:    saas-release
-            branches:   [ saas-release, saas-release-heroku ]
+            current:    release-ms-13
+            branches:   [ release-ms-13, saas-release, saas-release-heroku ]
```

Comment:
Will this make any more sense to the customer than saas-release? I am doubtful, but maybe it doesn't matter that much what we call it. What if we used a date instead?

Reply:
@kellyemurphy We're changing the links to point to `current`, so the URLs will hopefully make more sense.

Reply:
@kellyemurphy that said, perhaps our What's New needs to start referencing the milestone that an entry applies to. Doing so would allow customers to make the connection.